src/main/java/org/reactome/release/qa/check/HumanReactionsWithNonHumanComplexesWithHumanComponentsCheck.java

```diff
@@ -7,24 +7,24 @@
 import org.reactome.release.qa.common.QACheckerHelper;
 import org.reactome.release.qa.common.QAReport;
 
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
 
 /**
  * QA check that finds non-human Complexes that contain human Components, that are members of human, non-manually inferred ReactionlikeEvents.
  */
 
 @SliceQATest
 public class HumanReactionsWithNonHumanComplexesWithHumanComponentsCheck extends AbstractQACheck {
+
+    // Rather than overload multiple methods that use/don't use a skiplist, an empty List is used here.
+    private static List<String> EMPTY_SKIP_LIST = new ArrayList<>();
```

Comment:
Minor thing, but since this is in a class meant to be used as an instance now instead of a utility static method based class, the `static` qualifier could be removed.